DEVELOPMENT.md

```diff
@@ -66,13 +243,238 @@ You can then take this .vsix file and install the extension into your vscode ins
+
+Next start debugging the extension. Make sure the “Extension” debug configuration is currently selected. You can press the green run button or alternatively press F5. 
+
+![debug_loc](https://github.com/user-attachments/assets/3ad6d93c-5d9c-40c3-a8e4-a05fc83a8875)
+
+You may see this window pop up
+
+![webpack_not_fully_loaded](https://github.com/user-attachments/assets/9d4f45f2-cc81-40de-a456-2aecbc3df9b7)
+
+Click on debug anyway once yarn watch says webpack is fully compiled. 
+
+![webpack-fully-loaded](https://github.com/user-attachments/assets/5dd3202d-a853-4d9c-8b4a-48c381256d0f)
+
+A new VSCode window should’ve popped up with the sampleWorkspace opened. 
+
+![debug-window](https://github.com/user-attachments/assets/4d7e4cca-3172-4ab3-bd5c-52a5061bd353)
+
+Open the command palette with `Ctrl + Shift + P` and type in `Daffodil Debug`. Select the `Configure launch.json` option. 
+
+![config-launc-josn-command-pal](https://github.com/user-attachments/assets/d40f23c8-f47c-43ae-87ec-42ebb9c705c8)
+
+Your window should look like this. 
+
+![daf-debug-settings](https://github.com/user-attachments/assets/ae682d7e-c579-4ba0-b613-91e4f8667f93)
+
+Scroll down and check these following options
+
+![open_3_checkbox](https://github.com/user-attachments/assets/6ca42872-3060-4d16-a72a-a95c25e356e2)
+
+Click save
+
+![bottom-daf-debug](https://github.com/user-attachments/assets/d1c7e622-f2cb-43ca-9fa9-d128174fa54d)
+
+Click on the Run and Debug Icon. Wizard Config should show at the top. 
+
+![wiz-config](https://github.com/user-attachments/assets/aac7c1c3-1dd7-4219-b2ca-e053a10bae42)
+
+Then select a schema file and a data file. (Note, if you want to hard code where the images are, you’re able to change the DFDL and data file path in the Daffodil Configure launch.json window)
+
+![select-dfdl-schema](https://github.com/user-attachments/assets/17c6a731-2b53-45f9-9566-98efb1fb398e)
+
+![select-test-image](https://github.com/user-attachments/assets/14a8cd56-38d3-423e-9430-cf78d40a258b)
+
+Your window should look like the following. Note that you may have to move some tabs around. 
+
+![ide-debug](https://github.com/user-attachments/assets/dbc8a48e-b528-4861-8163-6c167c4d60fa)
+
+Here’s an example view of the data editor (OmegaEdit) with the schema on the left. 
+
+![ide-debug2](https://github.com/user-attachments/assets/9b982947-2f9c-45b0-8d6f-7b86eef1bacd)
+
+Here’s the infoset diff view. The infoset is the resulting output XML/JSON file. 
+
+![ide-debug4](https://github.com/user-attachments/assets/09e58522-e782-465f-b185-783eaf7ae592)
+
+#### Changing the sampleWorkspace folder name and location
+If you would like to specify a different name and/or location for the sampleWorkspace is, modify the following line in `.vscode/launch.json`:
```

Comment:
Consider changing "If you would like to specify a different name and/or location for the sampleWorkspace is" to "If you would like to specify a different name and/or location for the sampleWorkspace"